data/mods/MindOverMatter/snippets/snippets_new.json

```diff
@@ -146,18 +158,78 @@
+      },
+      {
+        "name": "GRAY DAWN vs. NOBLE HORIZON testing",
+        "id": "lab_file_clairsentience_04",
+        "text": "In the interests of testing Dr. Al-Masri's hypothesis, we assembled a group of NOBLE HORIZON and GRAY DAWN mathematicians and had them both attempt to conduct a simple number guessing game.  Melchior provided a six-digit number and both teams attempted to guess it.  GRAY DAWN were competitive when the number was overseen by a third party, as expected, but when both teams were provided with a terminal interface and instructed to enter their guesses, GRAY DAWN scored no better than chance.  On the other hand, NOBLE HORIZON consistently managed to guess at least three digits and successfully guessed the entire six-digit sequence twice (out of ten total sequences).  This disparity was maintained when both groups were placed in emtpy rooms, instructed to write down their guesses on a sheet of paper, and place it into a box before leaving the room.  Finally, NOBLE HORIZON attained a better than chance score on the final test, when both groups were instructed to guess the sequence before Melchior was asked to provide a sequence.\n\nThis leads to two conclusions.  The first is that the alarmists are wrong; Melchior is not secretly harboring a genius intellect and plotting to control XEDRA from within, it is a computer, capable of processing vast amounts of data and using netherum mathematics to process that data much faster than would otherwise be possible.  The second is that while GRAY DAWN requires a specific source from which to attain new information, NOBLE HORIZON does not.  Even in situations where no information transfer would be possible, such as the final two testing scenarios, NOBLE HORIZON still had a better-than-chance rate of correctly determining the sequence."
```

Comment:
```suggestion
        "text": "In the interests of testing Dr. Al-Masri's hypothesis, we assembled a group of NOBLE HORIZON and GRAY DAWN mathematicians and had them both attempt to conduct a simple number guessing game.  Melchior provided a six-digit number and both teams attempted to guess it.  GRAY DAWN were competitive when the number was overseen by a third party, as expected, but when both teams were provided with a terminal interface and instructed to enter their guesses, GRAY DAWN scored no better than chance.  On the other hand, NOBLE HORIZON consistently managed to guess at least three digits and successfully guessed the entire six-digit sequence twice (out of ten total sequences).  This disparity was maintained when both groups were placed in empty rooms, instructed to write down their guesses on a sheet of paper, and place it into a box before leaving the room.  Finally, NOBLE HORIZON attained a better than chance score on the final test, when both groups were instructed to guess the sequence before Melchior was asked to provide a sequence.\n\nThis leads to two conclusions.  The first is that the alarmists are wrong; Melchior is not secretly harboring a genius intellect and plotting to control XEDRA from within, it is a computer, capable of processing vast amounts of data and using netherum mathematics to process that data much faster than would otherwise be possible.  The second is that while GRAY DAWN requires a specific source from which to attain new information, NOBLE HORIZON does not.  Even in situations where no information transfer would be possible, such as the final two testing scenarios, NOBLE HORIZON still had a better-than-chance rate of correctly determining the sequence."
```

Reply:
Thank you for double-checking all of this text!